production_configs/srcdep_zen20/lstchain_config.json

```diff
@@ -0,0 +1,275 @@
+      "min_number_picture_neighbors":2,
+      "use_only_main_island":false,
+      "delta_time": 2
+    },
+    "dynamic_cleaning": {
+      "apply": true,
+      "threshold": 267,
+      "fraction_cleaning_intensity": 0.03
+    },
+  
+    "random_forest_energy_regressor_args": {
+      "max_depth": 50,
+      "min_samples_leaf": 2,
+      "n_jobs": 4,
+      "n_estimators": 150,
+      "bootstrap": true,
+      "criterion": "squared_error",
+      "max_features": "auto",
+      "max_leaf_nodes": null,
+      "min_impurity_decrease": 0.0,
+      "min_samples_split": 2,
+      "min_weight_fraction_leaf": 0.0,
+      "oob_score": false,
+      "random_state": 42,
+      "verbose": 0,
+      "warm_start": false
+    },
+  
+    "random_forest_disp_regressor_args": {
+      "max_depth": 50,
+      "min_samples_leaf": 2,
+      "n_jobs": 4,
+      "n_estimators": 150,
+      "bootstrap": true,
+      "criterion": "squared_error",
+      "max_features": "auto",
+      "max_leaf_nodes": null,
+      "min_impurity_decrease": 0.0,
+      "min_samples_split": 2,
+      "min_weight_fraction_leaf": 0.0,
+      "oob_score": false,
+      "random_state": 42,
+      "verbose": 0,
+      "warm_start": false
+    },
+  
+    "random_forest_disp_classifier_args": {
+      "max_depth": 100,
+      "min_samples_leaf": 2,
+      "n_jobs": 4,
+      "n_estimators": 100,
+      "criterion": "gini",
+      "min_samples_split": 2,
+      "min_weight_fraction_leaf": 0.0,
+      "max_features": "auto",
+      "max_leaf_nodes": null,
+      "min_impurity_decrease": 0.0,
+      "bootstrap": true,
+      "oob_score": false,
+      "random_state": 42,
+      "verbose": 0.0,
+      "warm_start": false,
+      "class_weight": null
+    },
+  
+    "random_forest_particle_classifier_args": {
+      "max_depth": 100,
+      "min_samples_leaf": 2,
+      "n_jobs": 4,
+      "n_estimators": 100,
+      "criterion": "gini",
+      "min_samples_split": 2,
+      "min_weight_fraction_leaf": 0.0,
+      "max_features": "auto",
+      "max_leaf_nodes": null,
+      "min_impurity_decrease": 0.0,
+      "bootstrap": true,
+      "oob_score": false,
+      "random_state": 42,
+      "verbose": 0.0,
+      "warm_start": false,
+      "class_weight": null
+    },
+  
+  
+    "energy_regression_features": [
+      "log_intensity",
+      "width",
+      "length",
+      "x",
+      "y",
+      "wl",
+      "skewness",
+      "kurtosis",
+      "time_gradient",
+      "leakage_intensity_width_2"
+    ],
+  
+    "disp_method": "disp_norm_sign",
+  
+    "disp_regression_features": [
+      "log_intensity",
+      "width",
+      "length",
+      "wl",
+      "skewness",
+      "kurtosis",
+      "time_gradient",
+      "leakage_intensity_width_2"
+    ],
+  
+    "disp_classification_features": [
+      "log_intensity",
+      "width",
+      "length",
+      "wl",
+      "skewness",
+      "kurtosis",
+      "time_gradient",
+      "leakage_intensity_width_2"
+    ],
```

Comment:
same above (though disp reconstruction is not important for source-dependent analysis)